src/core/cpuCO2.py

```diff
@@ -2,6 +2,14 @@
 from PyQt5.QtWidgets import QApplication
 import sys
 from core.gui import PlotWindowApp
+import yaml
+import os
+
+# Read YAML file 
+config_file_path = os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(__file__))), 'config', 'config.yml')
```

Comment:
I don't know how it works for windows and mac, but in linux usually the config files for all the apps are stored in a  standard directory at `/home/<user>/.config/<app_name>/config.yaml`. I think that in general it's better to provide a standard directory for the config file that's independent from the location of the script, so that people can store all the configs there. We can look for the most common config paths for windows and mac and put our config there.

Reply:
Oh. On Windows I've always seen configuration files inside a dedicated app folder (e.g. ./user/vscode/configs... or /user/.aws/configs.... What about Mac @giulianogiari?